src/Microsoft.DncEng.SecretManager/Readme.md

```diff
@@ -94,17 +88,44 @@ Each service has at least one settings file. These are expected to be named `set
 }
 ```
 
+## Functionality in the Builds (FR Look Here)
```

Comment:
If you want to make something searchable for FR folks it should maybe have "first responder" or similar nearby.

Reply:
I was planning on linking to this section from the FR wiki.